warehouse/admin/views/verdicts.py

```diff
@@ -0,0 +1,66 @@
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from paginate_sqlalchemy import SqlalchemyOrmPage as SQLAlchemyORMPage
+from pyramid.httpexceptions import HTTPBadRequest, HTTPNotFound
+from pyramid.view import view_config
+from sqlalchemy.orm.exc import NoResultFound
+
+from warehouse.malware.models import MalwareVerdict
+from warehouse.utils.paginate import paginate_url_factory
+
+
+@view_config(
+    route_name="admin.verdicts.list",
+    renderer="admin/malware/verdicts/index.html",
+    permission="moderator",
+    request_method="GET",
+    uses_session=True,
+)
+def get_verdicts(request):
+    try:
+        page_num = int(request.params.get("page", 1))
+    except ValueError:
+        raise HTTPBadRequest("'page' must be an integer.") from None
+
+    verdicts_query = request.db.query(MalwareVerdict).order_by(
```

Comment:
At some point being able to search or filter verdicts will probably be helpful. Our most complex search in admin is for Journals, see https://github.com/pypa/warehouse/blob/d756b8793e043d84179f160fe30107070f11b5b1/warehouse/admin/views/journals.py#L45-L63.

Though in this case it would probably be better to have pickers of some kind, to be able to filter by classification, confidence, and if it has been reviewed by an admin. We don't have any prior art for this in the warehouse admin though.

Reply:
Yes! That will definitely need to be added for this to be useful in production.